media-domain/package.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "jasmine": "^3.10.0",
```

Comment:
Moeten we deze dependencies nog pinnen?

Reply:
Misschien beter. Dit zijn domweg suggesties van intellij, ik heb er niet al te diep bij stil gestaan. Ik kijk er even naar.